docs/software-development-lifecycle.mdx

```diff
@@ -3,4 +3,40 @@ id: software-development-lifecycle
 title: Software Development Lifecycle
 ---
 
-TODO
+In general, a developer should a WIP (work in progress) limit of 1 story at a time (In Development through Complete). The developer claims ownership of that story through the entire development cycle.
```

Comment:
Add "have" or "maintain" for the following: "In general, a developer should [missing word] a WIP (work in progress) limit of 1 story at a time (In Development through Complete).